web/syncPoolHandler_elements.go

```diff
@@ -155,7 +157,7 @@ func (p *handlerPool) getElement(uid string) (*poolElement, bool, error) {
 
 		element = &poolElement{
 			uid:     uid,
-			handler: NewSyncUserHandler(uid, db, nil),
```

Comment:
Yup. If it is `nil` it will use the default config. I should actually change it to match the pysync server's